src/propTypes/albumTypes.js

```diff
@@ -0,0 +1,28 @@
+import PropTypes from 'prop-types';
+
+export const albumShape = PropTypes.shape({
+  _id: PropTypes.string.isRequired,
+  albumName: PropTypes.string.isRequired,
+  artistName: PropTypes.string.isRequired,
+  image: PropTypes.string.isRequired,
+  // custom validator function takes three arguments
+  releaseDate: (props, propName, componentName) => {
+    if (!/^\d{4}-\d{2}-\d{2}$/.test(props[propName])) {
```

Comment:
Currently, the backend returns a `releaseDate` that is a full date + time timestamp:

<img width="649" alt="Screenshot 2023-11-03 at 9 02 13 AM" src="https://github.com/Code-the-Dream-School/dd-prac-team4-front/assets/9094098/b291a9a2-0ba0-4f31-a5fc-9959f7d1cd6e">

Your regex is currently matching on just a date (4 digits, a dash, 2 digits, a dash, 2 digits, and then the end of the string).

You can use https://regex101.com/ to play around with regex to find the correct regex to match the expected strings:

<img width="904" alt="Screenshot 2023-11-03 at 9 03 40 AM" src="https://github.com/Code-the-Dream-School/dd-prac-team4-front/assets/9094098/c5ce993a-f791-4d3d-a0e2-3cb19c8efc94">

Reply:
<img width="541" alt="1" src="https://github.com/Code-the-Dream-School/dd-prac-team4-front/assets/95896159/16ca022f-5778-45c9-88f9-fc77a9492a63">

Reply:
@akosasante Can you check, if it's correct. Look like work😅

Reply:
👍